### ISSUE TITLE
Print shows empty page after every page

### ISSUE THREAD

**Issue by MickL** [mozilla/pdf.js#7739]
Im using a little customized viewer.js and viewer.html. When printing i get an empty extra page after every page. Do you have any thoughts what could cause this?

It works within original viewer example. beforePrint() is untouched. I also don't see any additional styles attached to #printContainer except for original viewer styles. HTML-output looks good, too:

<img width="217" alt="bildschirmfoto 2016-10-19 um 17 42 58" src="https://cloud.githubusercontent.com/assets/5103210/19526111/9bb1e8c8-9623-11e6-8da5-bc9aeb8c807e.png">

**Comment by yurydelendik** [comment]
Look for an issue in CSS -- something tells (e.g. padding or height) that canvas or div bigger than a page. We cannot help without complete example. Also provide more details. Closing as incomplete.

**Comment by MickL** [comment]
I got it working. What i did:
I commented out `afterPrint` function content in viewer.js so the printContainer gets not deleted. Then i pressed print (cmd + p) and clicked cancel. In developer tools i rendered media print css. Now i was able to see the print output and inspect the css. I did the same with the pdf.js-example and compared both.

What needed to be done for me: body and container of printContainer needs to have `height:100%` and `overflow:visible`.
